src/settings/ui-options-defaults.json

```diff
@@ -21,6 +21,20 @@
         }
       ]
     },
+    "panelOnActivate": {
```

Comment:
I think this could be a boolean property and represented as a checkbox. but we haven't really added support for that to our Options object yet.